awx_collection/test/awx/test_module_utils.py

```diff
@@ -17,18 +70,23 @@ def test_duplicate_config(collection_import):
         'tower_username': 'bob',
         'tower_config_file': 'my_config'
     }
+
+    class DuplicateTestTowerModule(TowerModule):
+        def load_config(self, config_path):
+            assert config_path == 'my_config'
```

Comment:
This `load_config` method is still called for other config file locations like `~/.tower_cli.cfg` even if the `tower_config_file` parameter is specified.

Looks like that might not so-much be an issue with the test, but an issue with the expected behavior. Is it expected to _combine_ parameters from the user config file with the specified config file? My current thinking is probably not, but I need others to weigh in. If not, the module utils should have a code change.